Kickstarter-iOS/Views/Controllers/PledgeTableViewController.swift

```diff
@@ -94,6 +94,18 @@ class PledgeTableViewController: UITableViewController {
       .observeValues { [weak self] project, pledgeTotal in
         self?.pledgeSummaryCell?.configureWith(value: (project, pledgeTotal))
       }
+
+    self.viewModel.outputs.dismissShippingRules
+      .observeForUI()
+      .observeValues { [weak self] in
+        guard
```

Comment:
View controller that presented something should also be responsible for dismissing it (it shouldn't be the responsibility of the presented vc)

Reply:
Yep. We could make this simpler and hold reference to the view controller but I thought this was less confusing than introducing another property...It's mostly just a best practice to explicitly say what we're dismissing coz otherwise it could lead to unexpected bugs.

Reply:
@dusi and I chatted IRL, we're going to revert this for consistency with the rest of the codebase for now and look to solve this with an agreed upon pattern going forward. One option is perhaps a dismiss closure that we can optionally set on our view controllers which we will hopefully have conforming to a protocol defining all of our programmatic UI functions in the near future.  That way at the time of presenting we can define how this VC should be dismissed.